setup.py

```diff
@@ -42,6 +42,9 @@
           'setuptools',
           'plone.rest >= 1.0a6',  # json renderer moved to plone.restapi
           'PyJWT',
+          'five.intid',
+          'plone.api',
```

Comment:
@erral the plone.api dependency is a problem. We did not finish our discussion about plone.api: https://github.com/plone/plone.restapi/issues/234

Personally I don't have a problem with introducing that dependency now. Though, we have to think about the future implications...